src/components/CustomAccordion.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { Accordion, Card, Row, Col } from 'react-bootstrap';
+import { useAccordionToggle } from 'react-bootstrap/AccordionToggle';
+import AccordionContext from 'react-bootstrap/AccordionContext';
+
+import Plus from '../assets/plus';
+import Minus from '../assets/minus';
+import '../index.css';
+
+class CustomAccordion extends React.Component {
+  render() {
+    function IconToggleAccordion({ largeHeader, children, eventKey, callback }) {
+      const currentEventKey = useContext(AccordionContext);
+      const toggleIconOnClick = useAccordionToggle(eventKey, () => callback && callback(eventKey));
+      const isCurrentEventKey = currentEventKey === eventKey;
+      const minusIcon = largeHeader ? <Minus large /> : <Minus />;
+      const plusIcon = largeHeader ? <Plus large /> : <Plus />;
+
+      return (
+        <Row className={`accordion-header ${largeHeader ? 'accordion-header-large' : ''}`} onClick={toggleIconOnClick}>
+          <Col xs="auto"> {isCurrentEventKey ? minusIcon : plusIcon} </Col>
+          <Col>{children}</Col>
+        </Row>
+      );
+    }
+    return (
+      <Accordion className="mb-3">
+        <div className="accordion-container">
+          <IconToggleAccordion as={Card.Header} eventKey="1" largeHeader={this.props.large}>
```

Comment:
nit: at the top of the render you can just do like `const { large, body } = this.props` to destructure so that you don't have to keep typing this.props throughout